alpha/apps/kaltura/modules/extwidget/actions/embedPlaykitJsAction.class.php

```diff
@@ -865,4 +901,54 @@ public function getRequestParameter($name, $default = null)
 		$returnValue = parent::getRequestParameter($name, $default);
 		return $returnValue ? $returnValue : $default;
 	}
+	
+	protected function saveCurrentBundleCacheKey($context)
+	{
+		//Avoid updating current cache hash on the ui conf if dynamic params affecting the result are being sent
+		$versions = $this->getRequestParameter(self::VERSIONS_PARAM_NAME);
+		if($versions)
```

Comment:
Most KMS related calls are using versions, so not sure why to remove this, assume that all the case coming from a website to load UI conf, may constantly come with versions

Reply:
As the list of versions sent by KMS is dynamic and can change I want to avoid constantly changing the hash values associated with the player. When sending versions this means there is no constant bundle hash and this is a use case I don't wont to handle.